spec/integration/user_show_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+
+RSpec.describe 'User Show', type: :feature do
+  before(:each) do
+    @user = User.create(
+      name: 'Promise',
+      bio: 'Fullstack Developer',
+      photo: 'http://promise.com',
+      post_counter: 2
+    )
+
+    @post1 = Post.create(
+      author: @user,
+      title: 'First post',
+      text: 'This is my first post',
+      comments_count: 1,
+      likes_count: 3
+    )
+
+    @post2 = Post.create(
+      author: @user,
+      title: 'Second post',
+      text: 'This is my second post',
+      comments_count: 2,
+      likes_count: 2
+    )
+  end
+  describe 'user show pages' do
+    describe 'a user show page' do
+      it 'displays the users profile photo' do
+        visit user_path(@user)
+        expect(page).to have_css("img[src*='http://promise.com']")
+      end
+    end
+
+    it 'displays the user name' do
+      visit user_path(@user)
+      expect(page).to have_content('Promise')
+    end
+
+    it 'displays the user bio' do
+      visit user_path(@user)
+      expect(page).to have_content('Fullstack Developer')
+    end
+
+    it 'displays the right Number of posts' do
+      visit user_path(@user)
+      expect(page).to have_content('2')
+    end
+
+    it 'displays all the available users posts' do
```

Comment:
Please add a test that asserts user can see the user's first 3 posts on `user#index`.